### PR TITLE
Add pinch testing parallel

### PR DESCRIPTION
**Pull request by joakim-hove**
Add parallell test of the MULTZ + PINCH::ALL situation

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by blattms** [comment]
Closing in favor of #2776.

**Comment by blattms** [comment]
Sorry, wrong PR closed.

File: tests/update_reference_data.sh [hunk 1]
```diff
@@ -130,7 +130,7 @@ tests[udq_pyaction]="flow udq_actionx PYACTION_WCONPROD udq_pyaction"
 tests[spe1_foam]="flow spe1_foam SPE1FOAM"
 tests[wsegsicd]="flow wsegsicd TEST_WSEGSICD"
 tests[bc_lab]="flow bc_lab BC_LAB"
-tests[pinch_multz_all]="flow pinch PINCH_MULTZ_ALL pinch_multz_all"
+tests[pinch_multz_all]="flow pinch PINCH_MULTZ_ALL_BARRIER pinch_multz_all"
```
Comment (blattms) [inline tests/update_reference_data.sh:133]: This looks strange. Isn't the barrier thing in #2780?

Comment (joakim-hove) [reply tests/update_reference_data.sh:133]: Yes - you are right; change on wrong branch ;-(

**Pull request closed without merging**